### PR TITLE
[grafana] added support for Grafana image renderer callback protocol customizat…

### PR DESCRIPTION
**Pull request by ethernoy**
Resolve https://github.com/grafana/helm-charts/issues/889

Signed-off-by: Ethern Su <ehaprime@gmail.com>

### CODE REVIEW

**Comment by zanhsieh** [comment]
@ethernoy 
Could you remove test.yaml? We already have some test under ci.

**Comment by ethernoy** [comment]
> @ethernoy Could you remove test.yaml? We already have some test under ci. Also please bump the chart version to latest. Thank you.

Thanks for noticing, this file was added by mistake. Removed it.

**Comment by zanhsieh** [comment]
@ethernoy 
Could you sign DCO ? The instruction should be on Details next to "DCO Action required after 1s — DCO" in this page below.

**Comment by ethernoy** [comment]
@zanhsieh Done

File: charts/grafana/README.md [hunk 1]
```diff
@@ -227,8 +227,9 @@ This version requires Helm >= 3.1.0.
 | `imageRenderer.hostAliases`                | image-renderer deployment Host Aliases                                             | `[]`                             |
 | `imageRenderer.priorityClassName`          | image-renderer deployment priority class                                           | `''`                             |
 | `imageRenderer.service.enabled`            | Enable the image-renderer service                                                  | `true`                           |
-| `imageRenderer.service.portName`           | image-renderer service port name                                                   | `'http'`                         |
+| `imageRenderer.service.portName`           | image-renderer service port name                                                   | `http`                           |
 | `imageRenderer.service.port`               | image-renderer service port used by both service and deployment                    | `8081`                           |
+| `imageRenderer.grafanaProtocol`            | Protocol to use for image renderer callback url                                    | `'http'`                         |
```
Comment (zanhsieh) [inline charts/grafana/README.md:232]: Please align with line 230 `http`

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Comment by zanhsieh** [comment]
@ethernoy 
Also the Chart version bump is needed. Thank you.

**Comment by ethernoy** [comment]
> @ethernoy Also the Chart version bump is needed. Thank you.

Both changes implemented.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**